src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -3745,18 +3745,21 @@ class CL0<T>
 " }, parseOptions: TestOptions.Regular.WithNullCheckingFeature());
 
             c.VerifyDiagnostics(
-                // (12,16): warning CS8620: Nullability of reference types in argument of type 'CL0<string?>' doesn't match target type 'CL0<string>' for parameter 'x' in 'void C.M1(ref CL0<string> x)'.
+                // (12,9): warning CS8619: Nullability of reference types in value of type 'CL0<string?>' doesn't match target type 'CL0<string>'.
```

Comment:
>// (12,9): warning CS8619: Nullability of reference types in value of type 'CL0<string?>' doesn't match target type 'CL0<string>'. [](start = 16, length = 130)

This looks like duplicate of the warning below.

Reply:
I see that this warning was reported before the change, but at a different location.

---
In reply to: [136656548](https://github.com/dotnet/roslyn/pull/21825#discussion_r136656548) [](ancestors = 136656548)